### PR TITLE
feat(web): add document details page

### PR DESCRIPTION
**Pull request by james-hill-kc**
## Describe your changes

add document details page

## Issue ticket number and link

BOAT-457

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [ ] I have performed a self-review of my own code
- [ ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [ ] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: appeals/web/src/server/appeals/appeal-documents/appeal-documents.controller.js [hunk 1]
```diff
@@ -1,26 +1,37 @@
...
+export const renderDocumentUpload = async (request, response, backButtonUrl, nextPageUrl) => {
 	const { appealId, documentId } = request.params;
-	const { currentFolder } = request;
+	const { currentFolder, errors } = request;
 
 	if (!currentFolder) {
 		return response.status(404).render('app/404');
 	}
 
+	if (!currentFolder.path) {
```
Comment (fabfab-kc) [inline appeals/web/src/server/appeals/appeal-documents/appeal-documents.controller.js:23]: Maybe we should log this to the logger instead... 
`import pino from '#/lib/logger.js';`

**Review by fabfab-kc — DISMISSED** [review]
Great work! Just a small comment about replacing a console.log with the `pino` logger.
Unfortunately, that data structure of document / redaction status is incorrect, but we'll try to minimise the changes when we'll do that.

File: appeals/web/src/server/appeals/appeal-documents/appeal-documents.controller.js (hunk 1 above)
Comment (james-hill-kc) [reply appeals/web/src/server/appeals/appeal-documents/appeal-documents.controller.js:23]: Oh I actually left this in by mistake, well spotted!

**Review by fabfab-kc — APPROVED** [review]
LGTM!

**Pull request merged**